### DISCUSSION TITLE
White label ixo-WebClient

### DISCUSSION THREAD

**Discussion by ig-shaun** [ixofoundation/ixo-webclient#472] [Ideas]
In preparation for Market Relayers hosting their own Internet of Impact Web Portals, using the ixo-WebClient, we would like to define the features which users would like to customise from a White Label implementation, within the following categories:

1. Branding and styling
2. Naming and localisation
3. Filters data model
4. Icon sets
5. Environment Parameters
6. App plugins for the Control Panel
7. AI Assistant Custom Actions and Custom Responses
8. Dashboard customisations
9. Performance Shields on the Control Panel
10. Custom Schemas for templates

Please add your suggestions and ask questions in this discussion thread.
If you have a specific feature request, please add this as a new issue.

**Comment by lohanspies** [comment]
@ig-shaun for now we would like to focus our attention on 1 to get a relayer up and running. Thereafter we can focus on 1-6.
@sciphergfx please let us know what is the effort and CI/CD process to achieve number 1.

**Reply by lohanspies** [reply]
As discussed we want the branding and styling to be decoupled from the rest of the code to ensure we can easily white label.

**Reply by sciphergfx** [reply]
Hi @lohanspies An approach is to pull the repo, create our own UI components (with styling) in a separate folder and reference components that we need from the code base. I think We'll be good overtime with the folder structure in tact. With regards the CI/CD process, i'll get back to you here, don't have any ideas now.
